Source/santad/Logs/EndpointSecurity/Serializers/ProtobufTest.mm

```diff
@@ -93,19 +94,35 @@ JsonPrintOptions DefaultJsonPrintOptions() {
   return options;
 }
 
-NSString *EventTypeToFilename(es_event_type_t eventType) {
```

Comment:
Note: Some of the diff here was due to the full serializers being implemented and tested. While the changes here to use variants are used yet in this PR, they will be used in the next PR. It became a bit too unwieldy to break this out so it was left in.